engine/engine_types.go

```diff
@@ -59,6 +59,7 @@ type Settings struct {
 	EnableExchangeRESTSupport      bool
 	EnableExchangeWebsocketSupport bool
 	MaxHTTPRequestJobsLimit        int
+	RequestMaxRetryAttempts        int
```

Comment:
This is supposed to replace the `RequestTimeoutRetryAttempts` but I've kept that around for the moment given it is a flag supported by the application

I need to verify that this (and the old flag) are working as expected

Reply:
Great! I'll go about removing the old config later today 👍

Reply:
Old setting/flag has now been removed